rush.json

```diff
@@ -993,7 +993,7 @@
       "packageName": "@rushstack/rush-serve-plugin",
       "projectFolder": "rush-plugins/rush-serve-plugin",
       "reviewCategory": "libraries",
-      "shouldPublish": false
+      "shouldPublish": true
```

Comment:
The other first party plugins are lockstepped with Rush since Rush embeds them for backwards compatibility. This plugin should only need to have its version bumped when the API surface changes, though that may be equivalent to being lockstepped with Rush.